Gemfile

```diff
@@ -32,6 +32,10 @@ gem 'jbuilder', '~> 2.5'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+gem 'factory_bot_rails' # #comment from besss: Needed so we can load fixtures for demos in production
+gem 'faker' # #comment from besss: Needed so we can load fixtures for demos in production
+gem 'ffaker'
```

Comment:
Could you move the above three gems to the development/test group.

Reply:
apparently if they are here, as you can see from the comments we can do demos easier somehow so I figured I leave them there. I can move them for now though if you would prefer